Assignment/solution_student/select_fun_part2/js/main.js

```diff
@@ -1,29 +1,55 @@
 // 1.	Slide the header up when it is clicked on.
+$("header").on("click", function(){
+	$("header").slideUp(400);
+});
 
 // 2.	Remove the purplebox class
+$(".box").removeClass("purplebox");
 
 // 3.	Hide all of the div’s in #row4
+$("#row4").hide();
 
 // 4.	Add a paragraph that says “I have been added” to all boxes with class “add-para” (Hint: you'll need to look up the append() method)
+$(".add-para").append("I have been added");
```

Comment:
You got it done right, but I would suggest appending the text wrapped in a `<p>` tag, because as it stand it's just plopping text in there with no containing element (http://d.pr/i/RLKS), and this is definitely a `<p>` if you ask me.
So something like:
```
$('.add-para').append('<p>I have been added</p>');
```